dashboard/app/helpers/levels_helper.rb

```diff
@@ -512,6 +512,7 @@ def blockly_options
 
     # Process level view options
     level_overrides = level_view_options(@level.id).dup
+    level_options['embed'] = level_options['embed'] || level_options['widgetMode']
```

Comment:
Is `level_options['embed']` used anywhere else? I wonder if overloading it to include widget mode could have unintended effects

Reply:
It is used in other places, but in this case, those side effects are things we want. Prior to this PR, I've been asking the ed team to use embed and widget on all levels where they want widget mode. Now, they won't need to check both boxes anymore and can just select widget mode. 

More detail: widget mode is essentially a stricter embed mode. We want all the functionality of embed along with additional functionality specific to widgetMode (i.e. removing the code/design/data buttons and the run/reset buttons). So in this case, it makes sense to overload embed for levels that use widget mode.